src/main/java/kitchenpos/product/domain/ProductRepository.java

```diff
@@ -0,0 +1,15 @@
+package kitchenpos.product.domain;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import java.util.List;
+
+public interface ProductRepository extends JpaRepository<Product, Long> {
+
+    @Query(value = " "
+            + " SELECT COUNT(p.id) "
+            + "   FROM Product  p "
+            + "  WHERE p.id IN (:ids)")
+    Integer countAllByIds(List<Long> ids);
```

Comment:
java8 이상 버전 부터는 javac flag -parameters 사용을 하면 해결이 되지만
해당 옵션은 각 개발 환경마다 설정해줘야 하는 부분이기에 `@Param`을 명시해줘도 괜찮지 않을까 합니다 😄 
```suggestion 
    Integer countAllByIds(@Param("ids") List<Long> ids);
```

Reply:
말씀 주신 부분에 동의합니다 !
공식 문서에서도  `@Query` 어노테이션 안에서 사용되는 파라미터의 경우 `@Param` 어노테이션을 사용 하는 것을 권장하고 있네요 😊
[Spring Data JPA - Reference. Using Named Parameters](https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#jpa.named-parameters)
피드백 주신 부분 반영하였습니다 !